dotenv/.env

```diff
@@ -0,0 +1,2 @@
+GREETING=hello world
+DO_NOT_OVERRIDE=overridden
```

Comment:
I suggest to move all of these files into `dotenv/testdata/` directory

Reply:
I've moved most of the test files to `./testdata`. I left `.env` and `.env.defaults` as they are used to test the default behaviour.